### PR TITLE
Documentation for HTTP proxy use

### LINKED ISSUE #358: Use of proxies needs documentation
I'll do a pull request with my suggested doco for how to use unauthenticated proxies.

Need to understand how to do 100% automated authenticated proxies #359

### PR DESCRIPTION
**Pull request by rwillmer**
To fix #358

### CODE REVIEW

File: docs/http-proxies.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+Using HTTP Proxies
+++++++++++++++++++++++++++++++++++++++++++++
+
+Unauthenticated proxies are simple, you need only configure
+the browser with the hostname and port.
+
+Authenticated proxies are rather more complicated, (see
+`RFC2617 <http://www.ietf.org/rfc/rfc2617.txt>`_)
+
+Using an unauthenticated HTTP proxy with FireFox
```
Comment (hltbra) [inline docs/http-proxies.rst]: Typo: "FireFox" vs "Firefox"

**Comment by hltbra** [comment]
I think this PR is a good start. It would be great to have more docs on different web drivers (chrome is a big deal).

Ideas on this @andrewsmedina?

**Comment by hltbra** [comment]
Matthew Goins (@mjgoins) recommends a different (simpler & easier) approach on the splinter-users mailing list, and I thin the docs should mention it: https://groups.google.com/forum/#!topic/splinter-users/rYxdWSd4jA8

    export http_proxy="http://proxy.example.com:3128"
    python my_python_script.py

**Comment by tarsisazevedo** [comment]
I'll merge this and update the doc with chrome usage.

**Pull request merged**